src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/NestedIfDepthCheck.java

```diff
@@ -42,6 +42,22 @@
  * <pre>
  * &lt;module name=&quot;NestedIfDepth&quot;/&gt;
  * </pre>
+ * <p>Valid Code Example:</p>
+ * <pre>
+ * if (true) {
+ *     if (true) {} // OK
+ *     else {}
+ * }
+ * </pre>
+ * <p>Invalid Code Example:</p>
+ * <pre>
+ * if (true) {
+ *     if (true) {
+ *         if (true) { // Nested if-else depth is 2 (max allowed is 1)
```

Comment:
Please also put violation comment.
like: `// violation, nested if-else depth is 2 (max allowed is 1)`
sorry it was confusing previously.
please do the same changes for all the other violations too.

Reply:
done

Reply:
done